pdb2sql/pdb2sqlcore.py

```diff
@@ -47,29 +48,33 @@ def __call__(self, **kwargs):
                     - no_chainID = ['A']
 
         Returns:
-            pdb2sql: an pb2sql instance 
+            pdb2sql: an pb2sql instance
 
         Examples:
             >>> sqldb pdb2sql('1AK4.pdb')
             >>> dbA = sqldb(chainID='A')
         """
-        pdb_data = self.sql2pdb(**kwargs)
-        return pdb2sql(pdb_data)
+
+        names = self._get_table_names()
+        if len(names) > 1:
```

Comment:
Is it possible for a pdb2sql instance have multiple table names?

Reply:
It is, and most of the method have to handle multiple table since `many2sql` is a subclass of `pdb2sql` But for clarity I would like to keep all the methods that operate over several tables (like intersect) in `many2sql`

Reply:
The `pdb2sql.__init__` allows only one pdb as input, so I think the `pdb2sql` instance will have only one table and its `__call__` does not to check number of tables. Though `many2sql`  is a subclass of `pdb2sql` but it has a reimplemented `__call__` to deal with multiple tables.

Reply:
you could do :
```python
db = pdb2sql('1AK4.pdb', tablename='1AK4')
db._create_table('3CRO.pdb', tablename='3CRO')
```

and then you'll have two tables in a `pdb2sql` instance. The `__call__` of `pdb2sql` does check for the number of table, print a warning if multiple tables are found and output the results of the first table only.

Reply:
Indeed, it's necessary to avoid the case using the private method.